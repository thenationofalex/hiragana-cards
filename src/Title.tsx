import React from 'react';
import {StyleSheet, Text, View} from 'react-native';

interface ITitle {
  title: string;
}

const Title = ({title}: ITitle) => {
  return (
    <View>
      <Text style={styles.base}>{title}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  base: {
    fontSize: 32,
    fontWeight: 'bold',
    marginBottom: 32,
    marginTop: 16,
    textAlign: 'center',
    textTransform: 'uppercase',
  },
});

export default Title;
