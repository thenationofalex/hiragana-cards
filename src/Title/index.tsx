import React from 'react';
import {StyleSheet, Text, View} from 'react-native';

interface ITitle {
  title: string;
}

const Title = ({title}: ITitle) => (
  <View>
    <Text style={styles.base}>{title}</Text>
  </View>
);

const styles = StyleSheet.create({
  base: {
    color: 'black',
    fontSize: 32,
    fontWeight: 'bold',
    marginBottom: 8,
    marginTop: 16,
    marginLeft: 44,
    textAlign: 'left',
  },
});

export default Title;
