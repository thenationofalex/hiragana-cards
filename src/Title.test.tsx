import 'react-native';
import React from 'react';
import {render} from '@testing-library/react-native';

import Title from './Title';

describe('Title', () => {
  it('Renders correctly', () => {
    const app = render(<Title title="Title" />);
    expect(app).toMatchSnapshot();
  });
});
