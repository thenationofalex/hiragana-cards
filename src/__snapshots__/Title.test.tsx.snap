// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 32,
        "fontWeight": "bold",
        "marginBottom": 8,
        "marginLeft": 44,
        "marginTop": 16,
        "textAlign": "left",
      }
    }
  >
    Title
  </Text>
</View>
`;
